app/Models/Trade.php

```diff
@@ -42,7 +42,7 @@ class Trade extends Model
     protected $filterableFields = ['requested_items', 'description'];
 
 
-    public function offers()
+    public function items()
```

Comment:
why did you change this back?

Reply:
I must have merged main into my branch earlier on so I used items() all throughout the feature. I'll refactor all of it before merging.